app/azure_logging.py

```diff
@@ -0,0 +1,62 @@
+import logging
+from typing import Optional
+
+from opencensus.ext.azure.log_exporter import AzureLogHandler
+from opencensus.trace import config_integration
+from opencensus.trace.samplers import AlwaysOnSampler
+from opencensus.trace.tracer import Tracer
+
+def telemetry_processor_callback_function(envelope):
+    envelope.tags['ai.cloud.role'] = 'model-api'
+    
+
+class ExceptionTracebackFilter(logging.Filter):
+    """
+    If a record contains 'exc_info', it will only show in the 'exceptions' section of Application Insights without showing
+    in the 'traces' section. In order to show it also in the 'traces' section, we need another log that does not contain 'exc_info'.
+    """
+    def filter(self, record: logging.LogRecord):
+
+        if record.exc_info:
+            logger = logging.getLogger(record.name)
+            _, exception_value, _ = record.exc_info
+            message = f"{record.getMessage()}\nException message: '{exception_value}'"
+            logger.log(record.levelno, message)
+
+        return True
+
+
+def initialize_logging(logging_level: int, correlation_id: Optional[str] = None) -> logging.LoggerAdapter:
```

Comment:
oh just noticed that this return value isn't used. if this is intended I would call as
```python
_ = initialize_logging(logging.INFO)
```
or make this void

Reply:
Done c953215056545073fc0ef60ef760c933d6ec0fb5